lib/adal.js

```diff
@@ -1187,7 +1141,7 @@ AuthenticationContext.prototype._cloneConfig = function (obj) {
     return copy;
 };
 
-AuthenticationContext.prototype._addClientId = function () {
+AuthenticationContext.prototype._addXClientParameters = function () {
```

Comment:
oh, sorry, i miss the x-client-sku. XClientParmeters is still not so clear to me, I have to look into the method for what it is adding.